src/stories/base/Button.stories.jsx

```diff
@@ -4,7 +4,7 @@ import { blockProp } from '@crcls/blocksui-sdk';
 
 // More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
 export default {
-  title: 'Primitives/Button',
+  title: 'Base Blocks/Button',
```

Comment:
Can you change these back to 'Primitive'?

Reply:
Updated